### PR TITLE
Made a few queryset lookup examples consistent.

### PR DESCRIPTION
**Pull request by Alexkane**

### CODE REVIEW

File: docs/ref/models/querysets.txt [hunk 1]
```diff
@@ -2562,7 +2562,7 @@ Case-insensitive starts-with.
 
 Example::
 
-    Entry.objects.filter(headline__istartswith='will')
+    Entry.objects.filter(headline__istartswith='Will')
```
Comment (timgraham) [inline docs/ref/models/querysets.txt]: `iendswith` is using lower case in both places so maybe it's better to change this SQL equivalent instead?

Comment (Alexkane) [inline docs/ref/models/querysets.txt]: I see what you mean.  I was following the convention in the documentation for `startswith`.  Would you prefer that it match `iendswith`?

Comment (timgraham) [inline docs/ref/models/querysets.txt]: I don't think it matters. Maybe 'Will' everywhere is simplest.

File: docs/ref/models/querysets.txt [hunk 2]
```diff
@@ -2582,11 +2582,11 @@ Case-sensitive ends-with.
 
 Example::
 
-    Entry.objects.filter(headline__endswith='cats')
+    Entry.objects.filter(headline__endswith='Cats')
```
Comment (timgraham) [inline docs/ref/models/querysets.txt]: Still not consistent... any reason for three examples to use "Will" and one use "Cats"?

**Pull request merged**